validation/lifecycle.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"errors"
+	"io/ioutil"
+
+	tap "github.com/mndrix/tap-go"
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/runtime-tools/validation/util"
+)
+
+func main() {
+	t := tap.New()
+	t.Header(0)
+
+	prestart := rspec.Hook{
+		Path: "/usr/bin/prestart",
```

Comment:
Where does `/usr/bin/prestart` come from?

Reply:
About this `path`, I thought it was created automatically when added.
Maybe this idea is wrong.
Do you have any suggestions for the `path`?

Reply:
> Do you have any suggestions for the `path`?

The echo on our BusyBox tarball?

Reply:
I'm writing a demo to show my idea, I'll link to you soom.